documentation/service_admin/con-address-plans.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-plans.adoc
+
+[id='con-address-plans-{context}']
+= Address plans
+
+Address plans specify the expected resource usage of a given address. The sum of the resource usage for all resource types determines the amount of infrastructure provisioned for an address space. A single router and broker pod has a maximum usage of one. If a new address requires additional resources and the resource consumption is within the address space plan limits, a new pod will be created automatically to handle the increased load.
+
+In the xref:con-address-space-plans-{context}[] section, the address space plan referenced two address plans: `small-queue` and `small-anycast`. These address plans are stored as `ConfigMaps` and are defined as follows:
```

Comment:
In the xref you want to type out what the context will be - so whatever you have context defined as should show in the link.

Reply:
done